test/cases/dml/load/load.test

```diff
@@ -0,0 +1,4 @@
+drop table if exists t1;
+create table t1(col1 bool,col2 int,col3 varchar, col4 date,col5 datetime,col6 timestamp,col7 decimal,col8 float);
+load data infile '$resources/load_data/jsonline.jl' into table t1 from_jsonline;
```

Comment:
How do you know this infile is a jsonline file?   Don't you need some kind of format = jsonline in the sql statement?

Reply:
Actually, let's add a comment -- 
1. format = jsonline jsondata = array    -- in this case, we are almost doing the same thing as CSV, loading each line, each element in the array is a column.
2. format = jsonline jsondata = struct   -- in this calse, we load using column names.

Reply:
Ah, you have from_jsonline -- do we use from_csv?   Or use format=csv?   Keep the same as csv.
I think you need more options so this from_jsonline thing is not enough.

Reply:
There is no from_csv or fmt=csv now. I think your thoughts are pretty good. Thx for your advice.